design-tokens/tokens/layer1/base.json

```diff
@@ -267,6 +297,30 @@
       }
     },
     "fontSize": {
+      "10": {
+        "value": "0.75rem",
+        "type": "fontSizes"
+      },
+      "20": {
+        "value": "0.875rem",
+        "type": "fontSizes"
+      },
+      "30": {
+        "value": "1rem",
+        "type": "fontSizes"
+      },
+      "40": {
+        "value": "1.25rem",
+        "type": "fontSizes"
+      },
+      "50": {
+        "value": "1.5rem",
+        "type": "fontSizes"
+      },
+      "60": {
+        "value": "2.125rem",
+        "type": "fontSizes"
+      },
```

Comment:
Why these new values?

Reply:
What's the logic behind the scale ? I remember for other stuff you tried to have the number 10 or 100 as the base value and then either smaller or bigger. Here the base value seems to be the 20? It will be the most used font-size everywhere right?

Reply:
ok make sense, anyway we shouldn't be using these tokens directly much